Removes lack of default-shaped borg icons and absence of slimtrap AI hologram icon, tweaks some borgs-related stuff

TobyThorne opened the pull request:
![standard droids](https://user-images.githubusercontent.com/45202681/80554411-dfe88200-89ee-11ea-99e0-725a53d26beb.PNG)

- Почему-то несмотря на наличие стандартного спрайта для боргов (весьма симпатичного, прошу заметить), у этого спрайта нет расцветок для различных модулей. Добавил вариации стандартного спрайта для всех типов борготы.
- Добавил шприцы боргам-хирургам. Без возможности колоть фарму сложные операции превращаются в невнятный цирк. По-хорошему, надо бы слить их с медиками в один модуль, ибо сейчас ни те, ни другие не могут соревноваться по эффективности с одним обычным медиком и брать их особого смысла нет.
- Добавил сварку, лампочкоменятель, мыло и декомпилятор материалов стандартному боргу. В данный момент брать его стоит только в надежде на малфа/ЕМАГающего тритора, ибо есворд. С этими инструментами можно делать хоть какие-то полезные действия - чинить рободрузей, разваривать стенки с застрявшими космонавтиками, оттирать бар от мяса и аутично менять лампочки.
- Добавил покрасчики пола и труб инжеборгам. 
- Добавил сервисным киборгам синтезатор. Синтезатор - мощь, синтезатор - сила. 
- Борги больше не могут тыкать в космонавтиков шприцы на харме и, как следствие, ломать их. Такой вот нерфоцирк.
- Борга снова могут чинить себя сваркой, но теперь - после трёхсекундной задержки, дабы не было гриновского самоотхила на бегу. После каждого карпа/блоба/летящей плитки бежать к мешкам с мясом и просить починить - такой себе увлекательный геймплей, особенно - на бегиннерсах. Выпил этой фичи - классический пример феерической бееразработочки.
- Добавлены тонкотрапский вариант стандартной голограммы для ИИ и голограмма "Катализатор" для малфа.
_На пике - стандартная голограмма, тонковариант и Катализатор._
![holos](https://user-images.githubusercontent.com/45202681/80557809-f3e5b100-89f9-11ea-9cbe-01a55fc56a0c.png)

- [x] Pull Request полностью завершен, мне не нужна помощь чтобы его закончить.
- [x] Я внимательно прочитал все свои изменения и багов в них не нашел.
- [x] Я запускал сервер со своими изменениями локально и все протестировал.
- [x] Я ознакомился c [Guide to Contribute](https://github.com/ChaoticOnyx/OnyxBay/blob/dev/docs/contributing.md).

Commit:
BigBadBorgsBupdate

- Added standard variations for all the modules
- Added syringes to the surgeon borgs
- Added welding tools, light replacers and soap bars to the standard borgs
- Added droppers and synthesizers to the service borgs
- Borgs can no longer stab people w/ syringes
- Borgs can repair themselves once agan, but after a 3s delay

Commit:
Holosexual Bonus

- Added Default-Slim AI hologram
- Added Catalyst malf hologram

koronoshka commented:
А спрайтики то реально симпатичные

Voldirs commented:
Раз уж на то пошло, убери охранный модуль для выбора раундстартом, но чтобы его можно было печатать в робототехнике там с золотом и серебром

TobyThorne commented:
@Voldirs 
Это уже лучше отдельным ПРом. Баланс боргов (секборгов - особенно) неплохо было бы основательно потрогать, но это надо обсуждать.

Voldirs commented:
Ну ладушки

TapKo4eB commented:
Пара замечаний:

Раз дал стандарт боргу лампочкоменятель, то дай маттер-декомпилер, дабы сломанные лампочки перерабатывать.

![image](https://user-images.githubusercontent.com/48761132/80573584-e4ba2f80-8a08-11ea-9342-414a5fef691f.png)
чё две пипетки у сервиса

koronoshka commented:
Первый коронный, второй похоронный.

Commit:
dropper fix

TapKo4eB commented:
маттер декомпилер стандартному боргу

Doster-d commented:
Было бы неплохо добавить мультитул научному боргу

MrMagick2104 commented:
> Было бы неплохо добавить мультитул научному боргу

Робоклешня научного борга, по-моему, может брать и использовать как мультитул, так и провода, разве нет?

Commit:
Update robot_modules.dm

- Added Matter Decompiler to the standard borg
- Enabled light replacer recharge for the standard borgs

Commit:
sorting robots.dmi

- Sorted all the sprites by types and jobs

Commit:
Engiborg update

- Added floor and pipe painters for the engiborgs

TobyThorne commented:
Попутно отсортировал это ёбаное безобразие.
![image](https://user-images.githubusercontent.com/45202681/80774185-13a5e200-8b7e-11ea-9a07-4ac7f74ec941.png)

TapKo4eB reviewed (approved):
нойс

LegatePritcher commented:
Извиняюсь за оффтоп, а кто этот молодой человек...
![image](https://user-images.githubusercontent.com/46104710/80792620-52539080-8bae-11ea-8f0e-95172be86229.png)

Poromoks commented:
Раз уж тут такое дело, просто оставлю иссуй #2192 тут. И да, клешня научборга не может подбирать мультитул (провода может), лучше уж его просто в инвентарь добавить. К слову, поддерживаю идею о том, что надо объединить медицинского и хирургического боргов в один модуль.

TobyThorne commented:
@dnivji @Moscowchief @Rodial-Admiral
![image](https://user-images.githubusercontent.com/45202681/81038431-cac89300-8ec7-11ea-9782-572f4a67b301.png)

dnivji commented:
Извиняюсь за слоупочество
АППРУВ

Moscowchief commented:
Аппрув. А идея с объединением медицинского и хирургического звучит интересно, может ее сразу и реализовать?

TobyThorne commented:
>А идея с объединением медицинского и хирургического звучит интересно, может ее сразу и реализовать?

Можно было бы, но есть проблемка с тем, что инвентарь будет на пол-экрана. Надо что-то хитрое с этим придумать. Так что, наверное, в следующий раз, вместе с балансом секборгов.

TobyThorne commented:
Опять Шрёдингер с плашками балуется >:(
![image](https://user-images.githubusercontent.com/45202681/81499019-2a6cd700-92ea-11ea-883b-5be5211b182c.png)

Animesku commented:
Ревью от кодеров, одобрено геймдизами.
